junitparser/junitparser.py

```diff
@@ -268,7 +273,7 @@ def __eq__(self, other):
         return super().__eq__(other)
 
 
-POSSIBLE_RESULTS = {Failure, Error, Skipped}
+FINAL_RESULTS = {Failure, Error, Skipped}
```

Comment:
should this even still be a global variable? its only used once and that usage could be simplified by removing the variable and use the content directly